internal/sink/matcher.go

```diff
@@ -172,10 +190,13 @@ func (m1 *MatcherV1) IsMatch(msg *wrp.Message) bool {
 		return false
 	}
 
-	for _, deviceRegex := range matcher {
-		if deviceRegex.MatchString(msg.Source) || deviceRegex.MatchString(strings.TrimPrefix(msg.Destination, "event:")) {
-			matchDevice = true
-			break
+	if matcher != nil{
```

Comment:
this is how old caduceus is doing this: https://github.com/xmidt-org/caduceus/blob/c9dc9a410874ee016a32f430f3f55b35e33f9ae2/outboundSender.go#L414